website/src/css/pages/_jobs.scss

```diff
@@ -245,6 +245,26 @@
         }
     }
 
+    .reviews-carousel {
+        height: 800px;
```

Comment:
It should be possible to clear out some more of these styles by using our `height-<whatever>` custom spacings (they also have breakpoints, which would handle the media queries below)

Reply:
Just thought I'd let you know @katjuell @st0nebraker, I [enhanced these](https://github.com/sourcegraph/about-replatform/pull/67/files#diff-d5059f7f3296770e92baba16d5e81763f80d55d19610817a103e1801725c641dR20-R38) a bit over in the replatform that could make this a lot easier on us until we get on Tailwind 😄 

So you should have these available to you if we port these over here:

breakpoint is `sm`, `md`, `lg`, `xl`
spacing is 100px–1200px

```scss
.h-#{$breakpoint}-auto {
    height: auto;
}
.h-#{$breakpoint}-#{$spacing} {
    height: #{$spacing}px;
}
.min-h-#{$breakpoint}-#{$spacing} {
    min-height: #{$spacing}px;
}
.max-w-#{$breakpoint}-#{$spacing} {
    max-width: #{$spacing}px;
}
```